emperor/support_files/js/color-editor.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * @author Yoshiki Vazquez-Baeza
```

Comment:
Just out of curiosity, are we adding these headers to all scripts? Just wondering as we remove them from the python code ...